thriftbp/client_middlewares.go

```diff
@@ -430,6 +430,9 @@ func getClientError(result thrift.TStruct, err error) error {
 		if typ.Field(i).Name == "Success" {
 			continue
 		}
+		if v.Field(i).IsZero() {
```

Comment:
note: there are both `IsZero` and `IsNil`, reading the doc I _think_ `IsZero` is more robust in this case.